Jenkinsfile

```diff
@@ -51,27 +51,57 @@ try {
             node('docker') {
                 deleteDir()
                 git 'https://github.com/jenkins-infra/azure.git'
+                /* Create an empty terraform variables file so that everything can
+                 * be overridden in the environment
+                 */
                 sh "echo '{\"prefix\":\"${tfPrefix}\"}' > ${tfVarFile}"
+
+                /*
+                 * Three following actions must be remove once terraform is greater than 0.8
+                 * Cfr INFRA-1288
```

Comment:
Terraform 0.9 introduce a new backend mechanism and the remote state must be pull with therraform client 0.8 then reinitialize with terraform client version 0.9.
Once jenkins-infra/azure:master is updated to 0.9 and deployed correctly, we can remove those lines